tools/library/tbc/lddecodemetadata.cpp

```diff
@@ -28,15 +28,19 @@
 const qint32 LdDecodeMetaData::LineParameters::sMinPALFirstActiveFrameLine = 2;
 const qint32 LdDecodeMetaData::LineParameters::sDefaultPALFirstActiveFieldLine = 22;
 const qint32 LdDecodeMetaData::LineParameters::sDefaultPALLastActiveFieldLine = 308;
+// Interlaced line 44 is PAL line 23 (the first active half-line)
 const qint32 LdDecodeMetaData::LineParameters::sDefaultPALFirstActiveFrameLine = 44;
+// Interlaced line 619 is PAL line 623 (the last active half-line)
 const qint32 LdDecodeMetaData::LineParameters::sDefaultPALLastActiveFrameLine = 620;
 const qint32 LdDecodeMetaData::LineParameters::sDefaultPALFieldHeightCheck = 313;
 
 // Default line parameters for NTSC decoding
 const qint32 LdDecodeMetaData::LineParameters::sMinNTSCFirstActiveFrameLine = 1;
 const qint32 LdDecodeMetaData::LineParameters::sDefaultNTSCFirstActiveFieldLine = 20;
 const qint32 LdDecodeMetaData::LineParameters::sDefaultNTSCLastActiveFieldLine = 259;
+// Interlaced line 40 is NTSC line 21 (the closed-caption line before the first active half-line)
 const qint32 LdDecodeMetaData::LineParameters::sDefaultNTSCFirstActiveFrameLine = 40;
+// Interlaced line 524 is NTSC line 263 (the last active half-line).
```

Comment:
I suppose "interlaced" actually means "frame" or "interleaved" in this context?  I've also seen this called psuedo-progressive numbering.  Not going to bikeshed it, but it's weird terminology IMHO.

Anyway, I haven't looked at PAL hard, but I don't think the NTSC comments are correct.  Unless numbering starts from 0?

Line numbering spreadsheet:
https://docs.google.com/spreadsheets/d/1pzVHFusLCI7kys2GzK9BTk3w7G8zcLxgHs3DMsurF7g/edit#gid=117149856

Reply:
I appreciate the feedback, but the change you're commenting on was simply me re-adding a comment that already existed prior to even my changes. So it should probably be given as a separate issue on Github, rather than on my pull request.

Reply:
I didn't meant to imply that action from you was required.  It's more a question for happycube or ats about how the numbering works (which was answered below).